app/Http/Controllers/UserController.php

```diff
@@ -61,4 +61,25 @@ public function update(UpdateUserRequest $request)
 
         return response($response, 200);
     }
+    public function getUsers(){
+        $users = User::all();
+        $email = $users->pluck('email');
+        $response = ['users' => $email];
+
+        return response($response, 200);
+    }
+    public function getUserData()
+    {
+        $user = Auth::user();
+        if (!$user) {
+            return response('not logged', 403);
+        }
+        if ($user->id == request()->route('id')) {
+            $response = ['user' => $user];
```

Comment:
point 7 of the task: for response use https://laravel.com/docs/7.x/eloquent-resources#writing-resources